### DISCUSSION TITLE
Account recovery

### DISCUSSION THREAD

**Discussion by Hatepoem** [chromiecraft/chromiecraft#3505] [General]
Greetings. I would like to ask to recover my account. The problem is, I forgot the password and __lost access to the email provider__ which I used for Chromiecraft, hence why I can't use "forgot password" feature.

I cannot verify my Discord account and therefore write here. Still here is the handle: Hatepoem#2491

 **REDACTED USER INFORMATION**

I will add any more identifying information that I am able. Please let me know if it is possible to recover my account.

**Comment by Nyeriah** [comment]
Hey,

For your own security, please avoid posting emails and other personal information in public platforms (such as the GitHub issue tracker or discussion board). I've redacted the email.

As for your issue, I'll ask you to kindly wait for one of the GMs to reply to this. 

Regards.

**Comment by Hatepoem** [comment]
Temper contacted me on Discord to verify the details. The issue is pending. Again, thank you.

**Comment by Nyeriah** [comment]
This subject has been handled in direct messages thus we're locking this thread. If you encounter any further issues feel free to contact us again.

Regards
